### PR TITLE
Mod/spr/update new main read me8810

### PR DESCRIPTION
**Pull request by niallriddell**
Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,16 @@
 <hr />
 <h1 style="color: red">IMPORTANT – PLEASE READ!</h1>
 
-<h3>On <strong>April 24, 2023</strong>, there will be a <strong>significant update</strong> to the React SDK <strong>main</strong> branch.</h3>
+<h3>On <strong>April 24, 2023</strong>, there has been a <strong>significant update</strong> to the React SDK <strong>main</strong> branch.</h3>.  The previous **main** branch has been moved to the **release/8.8.10** branch.
```
[the remark this answers is not in the document]
Comment (JEAxero) [reply README.md:6]: Also, change "to the .... branch" to "to this ... branch"

**Review by JEAxero — CHANGES_REQUESTED** [review]
A few minor suggestions

**Review by niallriddell — COMMENTED** [review]
Reviewed

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (niallriddell) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
+For those users that want to move to the new main branch **we strongly recommend** that you clone the latest main (known as release 8.23.10) into a new working folder locally. The existing SDK folder structure has significantly changed and may result in the deletion of folders in your local working copy.   Some recommendations on how to migrate your existing components to the new SDK are [available here](https://docs.pega.com/bundle/constellation-sdk/page/constellation-sdks/sdks/upgrading-sdk.html) along with a comprehensive set of documentation updates describing the [new capabilities](https://docs.pega.com/bundle/constellation-sdk/page/constellation-sdks/sdks/whats-new-sdk.html)
 
-**For any SDK user who wants to remain on the old main branch**, check out the following branch which will be created on April 24, 2023 before updates are made to the **main** branch:
+**For any SDK user who wants to remain on the old main branch**, check out this branch:
```
[the remark this answers is not in the document]
Comment (niallriddell) [reply README.md:11]: Done

**Review by niallriddell — COMMENTED** [review]
All changes have been incorporated

**Review by JEAxero — APPROVED** [review]
Looks good

**Pull request merged**